### PR TITLE
GSW-1839 helper to reset position's state

### PR DESCRIPTION
**Pull request by r3v4s**
## test
1. helper to reset position state

### CODE REVIEW

**Review by notJoon — APPROVED** [review]

**Comment by dongwon8247** [comment]
@r3v4s do you think we can distinguish the file names of test helpers more specifically? For example, I see [helper_test.gno](https://github.com/gnoswap-labs/gnoswap/blob/main/pool/helper_test.gno), [_helper_test.gno](https://github.com/gnoswap-labs/gnoswap/blob/main/pool/_helper_test.gno) in pool, but it can be confusing. If you don't have a good idea for naming, then maybe use `testhelper_0`, `testhelper_1` and add comments in the top lines.

**Comment by notJoon** [comment]
> @r3v4s do you think we can distinguish the file names of test helpers more specifically? For example, I see [helper_test.gno](https://github.com/gnoswap-labs/gnoswap/blob/main/pool/helper_test.gno), [_helper_test.gno](https://github.com/gnoswap-labs/gnoswap/blob/main/pool/_helper_test.gno) in pool, but it can be confusing. If you don't have a good idea for naming, then maybe use `testhelper_0`, `testhelper_1` and add comments in the top lines.

@dongwon8247 We've split it into two files to avoid conflicts, and will combine them into one ASAP.

**Review by onlyhyde — COMMENTED** [review]
@r3v4s 
We don't need the __TEST_0 code. We need to merge it into _helper_test. The contents of the _helper_test file should also be consolidated into _helper_test. Also, the reason for creating the helper as test code is to ensure that it is not used to manipulate the functionality of the contract. Here, we append `_` to the filename because GNO's test progression is folder-based, and the helper file for TEST should be executed before other test code.

If you have a better naming scheme, feel free to change it.

**Comment by onlyhyde** [comment]
@r3v4s 
Cleaned up related code in #422 pr. I recommend closing the current pr.

**Comment by r3v4s** [comment]
> @r3v4s Cleaned up related code in #422 pr. I recommend closing the current pr.

This pr is for position contract, not pool's position data.
IMHO we can let it open for now, and merge(or close) after we finish position contract refact

File: position/__TEST_0_INIT_VARS_HELPERS_test.gno [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	test1 std.Address = std.Address("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5")
+
+	fooPath string = "gno.land/r/onbloc/foo"
+	barPath string = "gno.land/r/onbloc/bar"
+	bazPath string = "gno.land/r/onbloc/baz"
+	quxPath string = "gno.land/r/onbloc/qux"
+
+	oblPath string = "gno.land/r/onbloc/obl"
+	// wugnotPath string = "gno.land/r/demo/wugnot" // from consts
+	// gnsPath string = "gno.land/r/gnoswap/v1/gns" // from consts
```
Comment (notJoon) [inline position/__TEST_0_INIT_VARS_HELPERS_test.gno:25]: Leftover?

Comment (r3v4s) [reply position/__TEST_0_INIT_VARS_HELPERS_test.gno:25]: It's just comment for where we're getting those two tokens path, unlikely others declared in that file.

Should we remove entire line?

Comment (notJoon) [reply position/__TEST_0_INIT_VARS_HELPERS_test.gno:25]: it will be fine

**Comment by onlyhyde** [comment]
> > @r3v4s Cleaned up related code in #422 pr. I recommend closing the current pr.
> 
> This pr is for position contract, not pool's position data. IMHO we can let it open for now, and merge(or close) after we finish position contract refact

okay, to avoid conflict, i will merge this pr

**Review by onlyhyde — APPROVED** [review]

**Pull request merged**